custom_model_runner/datarobot_drum/resource/components/Python/prediction_server/prediction_server.py

```diff
@@ -202,6 +205,21 @@ def predict_unstructured():
             finally:
                 self._post_predict_and_transform()
 
+            return (response, response_status)
+
+        @model_api.route("/chat/completions", methods=["POST"])
```

Comment:
@baekdahl , 
I think, we should also expose the `/v1/chat/completions`, in addition to the `/chat/completions`. Because it seems to be a part of OpenAI API standard. It's possible that in the future they'll change API to v2 and we'll have to be backwards compatible.

Predictions Gateway is configured to proxy either request:
```
# base_url="https://staging.datarobot.com/api/v2/deployments/669917cd46a0f93cd756e53d/"
base_url="https://staging.datarobot.com/api/v2/deployments/669917cd46a0f93cd756e53d/v1/"

client = OpenAI(
    base_url=base_url,
    api_key=os.environ.get('DATAROBOT_API_TOKEN'),
    http_client=httpx.Client(follow_redirects=True)
)



completion = client.chat.completions.create(
  model="generic_llm",
  messages=[
    {"role": "system", "content": "You are a stadup comic."},
    {"role": "user", "content": "Tell me a joke about programming"},
    {"content": "Why did the programmer quit his job?\n\nBecause he didn't get arrays! (get a raise)", "role": 'assistant'},
    {"role": "user", "content": "Can you write a longer joke on the same topic?"},
  ]
)

print(completion.choices[0].message)
```

Reply:
Makes sense. I've added the `v1` route as well.